gateway-service-definitions/src/main/resources/services/livy/0.4.0/rewrite.xml

```diff
@@ -45,4 +45,8 @@
     <rewrite template="{$frontend[url]}/yarn/nodemanager/node/containerlogs/{**}?{scheme}?host={$hostmap(host)}?{port}"/>
   </rule>
 
+  <rule dir="OUT" name="LIVYSERVER/livy/outbound/headers/ui" pattern="{scheme}://{host}:{port}/ui">
+    <rewrite template="{$frontend[url]}/livy/ui"/>
```

Comment:
Probably best to have trailing slash on this path.

Reply:
Added required changes.